index.html

```diff
@@ -22,6 +22,7 @@
     <h1 class="titleClass">Hello World</h1>
     <h2>This is the Caterpillar Kids Daycare Website!</h2>
   <br />
+  <img id="flyer" src="IMG_5569.jpeg" alt="flyer">
```

Comment:
can you change the image name of the file to poster.jpg or flyer.jpg
something that is recognizable thanks

Reply:
I changed the image name to flyer.jpeg. Sorry for do not use jpg, if I do that, then our flyer do not appear on the website.